pages/GSoC-19.md

```diff
@@ -5,4 +5,79 @@ permalink: /gsoc-19
 ---
 
 
-# Ideas will go here
\ No newline at end of file
+# Ideas for GSoC 2019
+----
+
+## Work on the Hydra Client
+### Idea:
+Currently the client is a proof of concept and only implements GET requests. We need a full fledged client that is in compliance with the spec. We also need to make sure that the client syncs with the server to keep it's data updated. This is not currently handled by the client. The client must also contain a battery of tests to ensure that a server is in compliance with the spec or not.

```

Comment:
`We also need to make sure that the client syncs with the server to keep it's data updated.`. 
The issue is in the hydrus repo https://github.com/HTTP-APIs/hydrus/issues/300.
I understand that the client needs to sync with the server but this feature has to implemented in the hydrus server first.